secure-ingress-cognito/envs/dev/values.yaml

```diff
@@ -0,0 +1,17 @@
+labels:
+  env: dev
+spec:
+  destination:
+    server: https://kubernetes.default.svc
+  source:
+    repoURL: https://github.com/aws-samples/eks-blueprints-workloads
+    targetRevision: main
+
+  # Thoses default values can be surcharged by Infrastructure as Code
+  env: dev
+  clusterName: eks-blueprint-cluster # the name of EKS cluster
```

Comment:
why cluster name is hard coded here? What if cluster name is different in reality?

Reply:
I can see this is how the other patterns are using as well. however, this will overridden if they are passing the cluster name.